pkg/templates/templates.go

```diff
@@ -84,6 +92,7 @@ var (
 type Config struct {
 	AdditionalIndentation uint
 	AESKey                []byte
+	DecryptionConcurrency uint8
```

Comment:
optional comment:  for readability and I would have probably grouped all encryption related config params to a sub-struct within the config struct.

Reply:
I like the idea. I'll do it in a followup PR to limit the changes in this PR.